util/export.py

```diff
@@ -18,21 +41,46 @@ def export(opt, cuda, model_in_file, model_out_file, opset_version, export_type)
         device = "cuda"
     else:
         device = "cpu"
-    dummy_input = torch.randn(
+
+    dummy_image = torch.randn(
         1, opt.model_input_nc, opt.data_crop_size, opt.data_crop_size, device=device
     )
+    dummy_inputs = [dummy_image]
+
+    if diffusion:
+        # Change model forward
+        if opt.model_type == "cm":
+            # max steps, mask
+            # TODO change inputs if necessary
+            sigmas = [80.0, 24.4, 5.84, 0.9, 0.661]
+            model = ConsistencyWrapper(model, sigmas)
+            dummy_inputs += [
+                torch.randn(
+                    1,
+                    opt.model_input_nc,
+                    opt.data_crop_size,
+                    opt.data_crop_size,
+                    device=device,
+                ),
+            ]
+        else:
+            # TODO add mask if necessary?
+            dummy_cond = torch.randn(1, opt.alg_diffusion_cond_embed_dim, device=device)
+            dummy_inputs.append(dummy_cond)
```

Comment:
Diffusion model export to onnx is not tested, maybe I should either remove it or test it with a diffusion model? In that last case, I need to know what inputs would be needed (mask, conditionning?)